api/shape/void.js

```diff
@@ -3,7 +3,9 @@ import { hasTypeGhost, hasTypeVoid } from '@jsxcad/geometry';
 import Shape from './Shape.js';
 
 export const voidFn = Shape.registerMethod(
-  'void',
+  ['void', 'hole'],
```

Comment:
Hole is nice

Reply:
Yeah, the critical problem is the inability to write void() :)